Loadsys/ruleset.xml

```diff
@@ -99,54 +130,21 @@
 
 
 	<!--
-	@TODO: These sniffs need to be reviewed to see if they duplicate
-	coverage already provided. The ones that should be kept need
-	tests written for them.
+	Pull in additional rules not previously covered by parent rulesets
+	(or explicitly excluded by them).
 	-->
-	<rule ref="Generic.Arrays.DisallowLongArraySyntax" />
-	<rule ref="Generic.CodeAnalysis.ForLoopShouldBeWhileLoop"/>
-	<rule ref="Generic.CodeAnalysis.ForLoopWithTestFunctionCall"/>
-	<rule ref="Generic.CodeAnalysis.JumbledIncrementer"/>
-	<rule ref="Generic.CodeAnalysis.UnconditionalIfStatement"/>
-	<rule ref="Generic.CodeAnalysis.UnnecessaryFinalModifier"/>
 	<rule ref="Generic.Commenting.Fixme"/>
-	<rule ref="Generic.Commenting.Todo"/>
-	<rule ref="Generic.Formatting.DisallowMultipleStatements"/>
-	<rule ref="Generic.Formatting.NoSpaceAfterCast"/>
 	<rule ref="Generic.Metrics.NestingLevel"/>
-	<rule ref="Generic.NamingConventions.CamelCapsFunctionName"/>
 	<rule ref="Generic.NamingConventions.ConstructorName"/>
-	<rule ref="Generic.NamingConventions.UpperCaseConstantName"/>
-	<rule ref="Generic.PHP.DeprecatedFunctions"/>
-	<rule ref="Generic.PHP.ForbiddenFunctions"/>
-	<rule ref="Generic.PHP.LowerCaseConstant"/>
-	<rule ref="Generic.PHP.NoSilencedErrors"/>
 	<rule ref="Generic.Strings.UnnecessaryStringConcat"/>
 	<rule ref="MySource.PHP.EvalObjectFactory"/>
-	<rule ref="PEAR.ControlStructures.ControlSignature"/>
 	<rule ref="PEAR.ControlStructures.MultiLineCondition"/>
-	<rule ref="PEAR.NamingConventions.ValidClassName"/>
-	<rule ref="PEAR.NamingConventions.ValidFunctionName"/>
-	<rule ref="Squiz.Arrays.ArrayBracketSpacing"/>
-	<rule ref="Squiz.Classes.LowercaseClassKeywords"/>
-	<rule ref="Squiz.Commenting.DocCommentAlignment"/>
-	<rule ref="Squiz.Operators.ValidLogicalOperators"/>
 	<rule ref="Squiz.PHP.DisallowMultipleAssignments"/>
 	<rule ref="Squiz.PHP.DisallowObEndFlush"/>
-	<rule ref="Squiz.PHP.DisallowSizeFunctionsInLoops"/>
```

Comment:
Yes, I verified using `phpcs -e`, which lists the rules in a set. Before and after these removals, our standard had 112 rules (I had to remove them slowly and re-check the count to see whether they were "still included" (by a parent) or not (and hence needed to be explicitly included in our own xml file.)